test/Unit/models/Mapper/ResultMapperTest.php

```diff
@@ -19,7 +19,7 @@
  *
  */
 
-namespace oat\taoResultServer\test\unit\models\Mapper;
+namespace oat\taoResultServer\test\Unit\models\nit\models\Mapper;
```

Comment:
Something really weird happened to this namespace.
Moreover, in order to truly follow PSR-4, it follow the same case as the file path, so it should be like that 👇.
```suggestion
namespace oat\taoResultServer\test\unit\models\Mapper;
```

Same applies to all the other namespaces.

Reply:
PHPStorm autocompletion happened :)
fixed. thanks

Reply:
if the case should be the same as in the path `Unit` should be uppercased.

Reply:
Huh, interesting.
Looks like at some point it was in lower case, and those sort of changes (changing a letter case of a directory) don't make into a diff, unless all the underlying files are modified. So I still have it in lower case locally.